source/renderer/app/components/wallet/settings/DelegateWalletButton.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React, { Component } from 'react';
+import { defineMessages, intlShape } from 'react-intl';
+import { Button } from 'react-polymorph/lib/components/Button';
+import styles from './DelegateWalletButton.scss';
+
+const messages = defineMessages({
+  label: {
+    id: 'wallet.settings.delegateWalletButtonLabel',
+    defaultMessage: '!!!Delegate',
+    description: 'Label for the delegate button on wallet settings',
+  },
+});
+
+type Props = {
+  disabled?: boolean,
+  onClick: Function,
+};
+
+export default class DelegateWalletButton extends Component<Props> {
+  static contextTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  render() {
+    const { disabled, onClick } = this.props;
+    const label = this.context.intl.formatMessage(messages.label);
+    return (
+      <Button
+        label={label}
+        disabled={disabled}
+        onClick={onClick}
```

Comment:
@yakovkaravelov maybe better function naming would be `onDelegate`

Reply:
👍